check/all_.py

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env python3
-
 import sys
 
-import general_superstaq.check
+import checks_superstaq as check
```

Comment:
Nope.  Maybe "this repo checks superstaq" is a nice play on words?  I am happy to make this `check` though, and even to consider different names entirely -- the naming collision between `./checks-superstaq/` and `./check/all_.py` (for example) is a :-1: for tab completion.

Reply:
haha i like "checks-superstaq" - i was mostly wondering if it would be more consistent to import it with `as checks` instead of `as check` (doesn't help with the tab-completion point though)

Reply:
Ah, I see.  Yeah `import checks` makes sense to me

Reply:
Oh I see I read your original comment backwards lol (swapping `check` and `checks`)